src/client/refactor/proxy.ts

```diff
@@ -39,8 +39,11 @@ export class RefactorProxy extends vscode.Disposable {
         // get line count
         // Rope always uses LF, instead of CRLF on windows, funny isn't it
         // So for each line, reduce one characer (for CR)
+        // But Not all Windows users use CRLF
         const offset = document.offsetAt(position);
-        return offset - position.line;
+
+        const winEols = getWindowsLineEndingCount(document.getText(), offset);        
+        return offset - winEols;
```

Comment:
@drzunny 
You are loading the entire document as a string (```document.getText()```), contrary to your comments.
>// In order to prevent the one-time loading of large files from taking up too much memory

You might want to use the VS Code api to read text from the document.